src/main/java/pl/isa/alphateam/Reservation.java

```diff
@@ -24,6 +25,15 @@ public Double getCostOfReservation() {
         return boat.getCostPerDay()*dayNo;
     }
 
+    public String getUUID(){
+        UUID uuid = UUID.randomUUID();
+        System.out.println("Twój kod potrzebny do rezerwacji to: " + uuid);
+        System.out.println("Wpisz go po zarejestrowaniu");
+        return null;
```

Comment:
Jeżeli nie zamierzasz nic zwracać, to przydałoby się zrobić metodę void, ale wtedy nie powinna nazywać się getUUID :)

Proponuję zrobić jedną metodę, która generuje kod i nazywa się getRegistrationCode, a w środku może generować UUID, a druga metoda żeby miała showRegistrationCode, która wyświetla informację użytkownikowi odnośnie kodu.